source/usr/bin/mcvirt

```diff
@@ -27,11 +27,8 @@ from mcvirt.auth import Auth
 
 if __name__ == "__main__":
 
-    parser_object = Parser()
-
     try:
-        # Ensure that the user is running MCVirt as root/sudo
```

Comment:
Is this functionality exported to the parser or are we removing it? I.e., can we now run mcvirt without sudo access?